### PR TITLE
Removes free insuls from loadout menu

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
haha FUCK

## How This Contributes To The Skyrat Roleplay Experience
No free insuls.

## Changelog

:cl:
del: Yellow gloves (loadout) no longer insulate
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
i wonder how long this has been sitting around

**Comment by Stalkeros2** [comment]
It'd be way better for yellow-wearing people (hey there gang) if you'd actually make them, you know, not insulated instead of removing them outright.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm [hunk 1]
```diff
@@ -60,11 +60,6 @@ GLOBAL_LIST_INIT(loadout_gloves, generate_loadout_items(/datum/loadout_item/glov
 	name = "Red Gloves"
 	item_path = /obj/item/clothing/gloves/color/red
 
-/datum/loadout_item/gloves/yellow
-	name = "Yellow Gloves"
-	item_path = /obj/item/clothing/gloves/color/yellow
```
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm]:
```suggestion
	name = "Budget Insulated Gloves"
	item_path = /obj/item/clothing/gloves/color/fyellow
```

Comment (Paxilmaniac) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm]: budgets can still be perfectly insulated

Comment (tf-4) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm]: fucking

Comment (tf-4) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm]:
```
/obj/item/clothing/gloves/color/faker_yellow
	name = "yellow gloves"
	desc = "A pair of yellow gloves."
	icon_state = "yellow"
	inhand_icon_state = "ygloves"
```

Comment (Zonespace27) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm]: Went ahead and changed how i did it

File: modular_skyrat/master_files/code/modules/clothing/gloves/color.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/obj/item/clothing/gloves/color/ffyellow // EXTRA fake, for the loadout
+	name = "yellow gloves"
+	desc = "At first glance, these look like insulated gloves, but... is that crayon?"
+	siemens_coefficient = 1
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/gloves/color.dm]: gloves have a coefficient of 0.5 (unless you intentionally want them worse which is funny)

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
Don't make a new file bro

**Comment by Zonespace27** [comment]
> Don't make a new file bro

alright done

**Comment by Iamgoofball** [comment]
>he didn't know the color yellow is naturally insulating

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**